inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -858,6 +865,14 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         return res;
     });
 
+    addSpecificCreator({"Elu"}, [](const std::shared_ptr<::ngraph::Node>& node,
+                                                 const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "elu", details::convertPrecision(node->get_output_element_type(0))};
```

Comment:
Hmm, why do you type elu type from lowercase?
If it is a typo, I think we shouldn't have a specific layer creator for the Elu op.

Reply:
Specific creator is provided due to model type difference between IR v10 and v5 (type="Elu" vs type="elu").